File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/payments/ResponsePaymentDto.java
package ps.com.viajeros.dtos.payments;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ResponsePaymentDto {

    private Long idPayment;
    private String paymentId;
    private String status;
    private String externalReference;
    private String paymentType;
    private String merchantOrderId;
    private Long idPasajero;
    private String nombreCompletoPasajero;
    private String cuilPasajero;
    private String cbuPasajero;
    private LocalDateTime fechaPago;

    // Getters y setters para cada propiedad
}

File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Payments/ResponsePaymentDto.ts
// src/app/models/response-payment-dto.ts

export interface ResponsePaymentDto {
    idPayment: number;
    paymentId: string;
    status: string;
    externalReference: string;
    paymentType: string;
    merchantOrderId: string;
    idPasajero: number;
    nombreCompletoPasajero: string;
    cuilPasajero: string;
    cbuPasajero: string;
    fechaPago: Date;
}
